mitodl/open-discussions
# Add models to store id information for posts, channels and comments

#### What are the relevant tickets?
Fixes #730 

#### What's this PR do?
Adds models to store id information for posts, channels, and comments. The functions which create new channels, posts and comments and the reindex management command will be updated to also update these tables.

#### How should this be manually tested?
Create a new channel, comment and post. Verify that there is matching information in `Channel` `Comment` and `Post` models.

Run `./manage.py recreate_index`. Verify that all posts, channels and comments have objects associated with them.
